api/user/index.ts

```diff
@@ -4,6 +4,9 @@ import { getConfigValue } from '../configuration'
 import { CASE_SHARE_PERMISSIONS, SESSION_TIMEOUTS } from '../configuration/references'
 
 export async function getUserDetails(req, res: Response, next: NextFunction) {
```

Comment:
Can you add a return type function `getUserDetails()`

Reply:
updated as mentioned